src/views/partials/breaches.js

```diff
@@ -138,35 +118,27 @@ export const breaches = data => `
     <span>${getMessage('column-detected')}</span>
   </header>
   ${createBreachRows(data.breachesData)}
-  <div class="no-unresolved-breaches-message">
-    <div class="no-breaches-message">
+  <template class='no-breaches'>
+    <div class="zero-state no-breaches-message">
       <img src='/images/breaches-none.svg' alt='' width="136" height="102" />
-      <h2>
-        ${getMessage('breaches-none-headline')}
-      </h2>
-      <p>
-        ${
-          data.breachesData.verifiedEmails.map(account => {
-            return `<span data-email="${account.email}" ${account.primary ? '' : 'hidden'}>${getMessage('breaches-none-copy', { email: `<b>${account.email}</b>` })}</span>`
-          }).join('')
-        }
+      <h2>${getMessage('breaches-none-headline')}</h2>
+      <p>${getMessage('breaches-none-copy', { email: '<b class="current-email"></b>' })}</p>
```

Comment:
Haven’t thought of doing it this way. In case we adopt this as a pattern in multiple places we should think of a higher order util that we can use for injecting string dynamically.